python/src/opendp/extras/polars/__init__.py

```diff
@@ -252,12 +367,33 @@ def sum(self, bounds: tuple[float, float], scale: float | None = None):
         * Applying noise to the sum
 
         >>> import polars as pl
-        >>> pl.col('numbers').dp.sum((0, 10))
-        col("numbers").clip([...]).sum()...:noise([...])
+        >>> context = dp.Context.compositor(
+        ...     data=pl.LazyFrame({"visits": [1, 2, None]}),
+        ...     privacy_unit=dp.unit_of(contributions=1),
+        ...     privacy_loss=dp.loss_of(epsilon=1.),
+        ...     split_evenly_over=1,
+        ...     margins={(): dp.polars.Margin(max_partition_length=5)}
+        ... )
+        >>> query = context.query().select(pl.col("visits").fill_null(0).dp.sum((0, 1)))
+        >>> query.release().collect()
+        shape: (1, 1)
+        ┌────────┐
+        │ visits │
+        │ ---    │
+        │ i64    │
+        ╞════════╡
+        │ ...    │
+        └────────┘
+
+        Output is noise added to two due to each value being clipped to (0, 1).
         """
         return self.expr.clip(*bounds).sum().dp.noise(scale)
 
-    def mean(self, bounds: tuple[float, float], scale: tuple[float, float] = (1.0, 0.0)):
+    def mean(
+        self,
+        bounds: tuple[float, float],
+        scale: tuple[float | None, float | None] = (None, None),
```

Comment:
Is there a unit test (doesn't belong in doctest) that checks behavior if either the first or second element in scale is `None`?

Reply:
Nope, but it just defaults to one